src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String report, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file: " + filePath, e);
+        }
+    }
```

Comment:
The FileWriterService implementation should not hardcode the file path. It's better to receive the file path as a parameter to the method, which you've done correctly. However, make sure that the file path comes from a configuration or directly from the user input in the main class, avoiding hardcoding it anywhere else in the application.